eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -116,7 +116,38 @@ stages:
                     PackageName: ${{artifact.name}}
                     ServiceName: ${{parameters.ServiceDirectory}}
                     ForRelease: true
-
+          - job: InstallationCheck
+            displayName: "Installation Check"
+            variables:
+              - template: ../variables/globals.yml
+            steps:
+              - download: current
+                artifact: ${{parameters.ArtifactName}}-signed
+              - pwsh: |
```

Comment:
I think the script could be simplified/made easier to read by setting the current directory at the template level:

```
- pwsh: ...
   workingDirectory: $(Pipeline.Workspace)
```